specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/virtualMachineImage.json

```diff
@@ -925,6 +978,25 @@
       ],
       "description": "Describes a Virtual Machine Image."
     },
+    "VmImagesInEdgeZoneListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VirtualMachineImageResource"
+          },
+          "description": "The list of VMImages in EdgeZone"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URI to fetch the next page of VMImages in Edgezone. Call ListNext() with this URI to fetch the next page of VmImages."
+        }
+      },
+      "required": [
```

Comment:
let's remove this tag since this is a "result" object.

Reply:
Sorry, remove which property exactly?

Reply:
required portion